tests/vmi_multus_test.go

```diff
@@ -797,11 +797,18 @@ var _ = Describe("[Serial]SRIOV", func() {
 			startVmi(vmi)
 			waitVmi(vmi)
 
-			vmi, err := virtClient.VirtualMachineInstance(vmi.Namespace).Get(vmi.Name, &metav1.GetOptions{})
-			Expect(err).NotTo(HaveOccurred())
-
-			interfaceName, err := getInterfaceNameByMAC(vmi, mac)
-			Expect(err).NotTo(HaveOccurred())
+			vmiNamespace := vmi.Namespace
+			vmiName := vmi.Name
+			var interfaceName string
+			Eventually(func() error {
+				var err error
+				vmi, err = virtClient.VirtualMachineInstance(vmiNamespace).Get(vmiName, &metav1.GetOptions{})
+				if err != nil {
```

Comment:
nit: maybe use vmi.Namespace and vmi.Name straight away? I don't see you use them elsewhere
then you can delete  `vmiNamespace := vmi.Namespace` and `vmiName := vmi.Name`

Reply:
The reason i didnt, is that in case the Get fails, it will override the vmi, and we can lose the vmi Name and Namespace

Reply:
At the end removed it, not part of this PR, and if it will happen we should see it and fix it in a global problem
(after discussing with Edy about it)